webui/react/src/components/InlineEditor.tsx

```diff
@@ -63,8 +63,11 @@ const InlineEditor: React.FC<Props> = ({
   const save = useCallback(async (newValue: string) => {
     if (onSave) {
       setIsSaving(true);
-      const err = await onSave(newValue);
-      if (err !== null) {
```

Comment:
After discussion with Emily, I support making this `if (err != null) {` and no other changes to this InlineEditor file

Reply:
Ok great! With that change, the test "preserves input when focus leaves" passes, whereas before it failed.

Reply:
great!  It turns out I was wrong about no other changes in the file
as you pointed out, the `Promise<void>` type should also be updated here.